CHANGELOG.md

```diff
@@ -1 +1,30 @@
+# 0.6.0
+
+## Prelude
+
+Added new features and commands, such as:
+
+- 'subgit delete'
+- 'subgit import'
+- 'subgit reset'
+- 'subgit clean'
+
+## General fixes
+
+- Added error handling for 'subgit pull'.
+	- If subgit config file has a repo with `branch: `, the command exits
+
```

Comment:
Indent error? and generally there should not be blank lines as that would make a dent in the list and make two lists